flake.nix

```diff
@@ -138,15 +138,10 @@
               else
                 RUN_ID=""
               fi
-
-              # get tempfile for telegraf config
-              TELEGRAF_CONFIG=$(mktemp --suffix ".conf")
-              cp ./telegraf/telegraf.local.conf "$TELEGRAF_CONFIG"
-
-              sed --in-place "s/run_id = \"\"/run_id = \"$RUN_ID\"/" "$TELEGRAF_CONFIG"
+              export RUN_ID
```

Comment:
We can set RUN_ID in the telegraf config via env instead of creating a temp config